### DISCUSSION TITLE
Problems to open a previous project

### DISCUSSION THREAD

**Discussion by Mapachosa** [thegetty/quire#279] [Get Help] [answered]
Hi, 
I have a problem opening a previous project in Quire. When I open the project from the terminal I get this message.

MacBook-Pro-de-Archivo:quire1 mapachosa$ cd
MacBook-Pro-de-Archivo:~ mapachosa$ cd quire1
MacBook-Pro-de-Archivo:quire1 mapachosa$ quire preview
(node:70706) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'config.yml'
    at Object.openSync (fs.js:497:3)
    at Object.readFileSync (fs.js:393:35)
    at B (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:24620)
    at n.value (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:35619)
    at new n (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:34603)
    at /usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:50209
    at new Promise (<anonymous>)
    at e.value (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:50175)
    at emit (events.js:400:28)
    at Command.<anonymous> (/usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js:2:122529)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:70706) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:70706) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

I have downloaded node version 14.18.1 and tried creating a new project and it worked normal but when I want to open the old project it rejects me. Can you help me please? I don't know if I'm doing something wrong.
thank you

**Comment by geealbers** [comment, marked as answer]
Hi @Mapachosa. It looks like the project you're trying to preview is called `quire1`, is that correct? If so, can you open that directory in your Finder and verify that it is has the files like the in the screenshot below, including the config.yml file?

(The 'quire1` directory will be in your Home directory, which you can find on a Mac by pressing Shift-Command-H in any Finder window.)

<img width="660" alt="Screen Shot 2022-03-29 at 2 21 15 PM" src="https://user-images.githubusercontent.com/7796401/160709270-bcfa999a-816b-4381-a37d-eb7031b2780f.png">

**Reply by Mapachosa** [reply]
Hi @geealbers. 
Thank so much for you quick response.
Yes, my folder is called Quire1. I tell you that this project has been worked on by another person and we want to work on it together. It even has another localhost.
First, I thought it was a quire installation problem, so I created a new project and it worked normally, but with this project when doing the preview it doesn't work.
the folder "quire1" looks like that

<img width="1037" alt="Screen Shot 2022-03-30 at 10 08 58" src="https://user-images.githubusercontent.com/90414710/160870374-ccdc1fe0-0fdc-4513-bed8-bc9c2aa924c2.png">

**Reply by geealbers** [reply]
@Mapachosa, that's all really helpful information, I see the problem. You'll need to remove the " copie" text from all of those file names, so that they look like this:

```
bin
CHANGELOG.md
config
config,yml
content
data
README.md
resources
site
static
themes
```

Hopefully, the sub-folders and sub-files within these top-level items don't also have " copie" in the filenames as well. That would also break the preview process and there are so many folders and files I think it would be very hard to manually rename them all.

Once you have them renamed though, because it's a copy of project, you'll also need to run `quire install` in your Terminal before you run `quire preview`. You'll only need to do the install once. After that preview should work like it does in the other project you started. 

So here are the commands you'd run when you first open Terminal:

```
cd quire1
quire install
quire preview
```

**Reply by Mapachosa** [reply]
@geealbers I did what you told me, but doesn't generate the preview. 
The version of Node es 14.18.1

<img width="650" alt="Screen Shot 2022-03-30 at 13 09 20" src="https://user-images.githubusercontent.com/90414710/160903138-c3ef2260-ad88-45cf-b761-a6be0936c3ee.png">

**Reply by geealbers** [reply]
Hmm. Can you try deleting the `themes/default/node_modules` directory, and then running the `quire install` and `quire preview` commands again? If it's still not working, try `quire preview --verbose` as that may give us some more Error information to go on.

**Reply by Mapachosa** [reply]
Hi @geealbers ,
I tell you to follow the steps and perfectly perfect at localhost:1313, I want to thank you for your patience and support. :) 
I would like to ask you how it could be done to work in a different localhost, as I told you this project is being developed by two people, however we live in different countries. For this occasion, we have first copied the entire project to a drive and then downloaded it, but we would like to work on it simultaneously without the need to upload and download the project. Is there any way to work it that way?

**Reply by Erin-Cecele** [reply]
Hi @Mapachosa I think GitHub may provide a good solution for you that is how we handle multiple people working on a single project at the Getty. Please read this section of your documentation: https://quire.getty.edu/documentation/github/ and let me know if that is of interest. If it is, we can arrange a time to hop on a call and do a quick GitHub tutorial.
